src/hydrusvideodeduplicator/dedup.py

```diff
@@ -237,6 +238,29 @@ def clear_search_cache() -> None:
         except OSError:
             rprint(f"[red] Database does not exist. Cannot clear search cache.")
 
+    # Updates the 'farthest_search_index' of all videos in the database that have an index greater than the length of
+    # the database, reducing them down to the length of the database.
+    @staticmethod
+    def update_search_cache(new_total: int | None = None) -> None:
+
+        if new_total is not None and new_total < 1:
+            raise Exception(f"passed in total must be greater than 0, but received {new_total}")
+        try:
```

Comment:
No need to try/except. If something that would throw an exception happens here the program should probably crash anyway since the rest of the program shouldn't and won't be recoverable e.g. db is modified externally or something like that. Also if an exception is raised inside the function then you can't catch it outside the function unless you reraise it, so if another function calls this one and wants to catch it an error it can't.

I abuse exceptions too much in the rest of the code and just realized it.